nodes/src/nodes/tool_git/IInstance.py

```
@@ -0,0 +1,470 @@
+                author=a.get('author') or None,
+                since=a.get('since') or None,
+                until=a.get('until') or None,
+            )
+        if name == 'git.show':
+            return git.show(ref=a['ref'])
+        if name == 'git.diff':
+            return git.diff(
+                ref_a=a.get('ref_a') or None,
+                ref_b=a.get('ref_b') or None,
+                path=a.get('path') or None,
+                staged=self._bool_arg(a, 'staged', False),
```

Ruff (0.15.12)

[warning] Boolean positional value in function call

(FBT003)